Dockerfile.simple

```diff
@@ -42,14 +42,14 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 		vim-common \
 	&& rm -rf /var/lib/apt/lists/*
 
-# Install runc, containerd, tini and docker-proxy
-# Please edit hack/dockerfile/install/<name>.installer to update them.
-COPY hack/dockerfile/install hack/dockerfile/install
-RUN for i in runc containerd tini proxy dockercli; \
-		do hack/dockerfile/install/install.sh $i; \
-	done
-ENV PATH=/usr/local/cli:$PATH
+# install dependencies as linked context from main Dockerfile to avoid
+# deduplication. see docker-bake.hcl for more info
+COPY --link --from=tini        /out/ /usr/local/bin/
+COPY --link --from=runc        /out/ /usr/local/bin/
```

Comment:
Quick comment; we should probably keep the `.simple` one to be compatible with classic builder (and other builders)

Reply:
> classic builder

Not sure I understand why?